src/beacon/drand.rs

```diff
@@ -47,8 +48,9 @@ pub enum DrandNetwork {
 #[derive(Clone)]
 /// Configuration used when initializing a `Drand` beacon.
 pub struct DrandConfig<'a> {
-    /// URL endpoint to send JSON HTTP requests to.
-    pub server: &'static str,
+    /// Public endpoints of the drand service.
+    /// See <https://drand.love/developer/http-api/#public-endpoints>
+    pub servers: Vec<&'static str>,
```

Comment:
Could we have it adequately typed on this level already?

Reply:
Could you elaborate? Do you suggest using `Vec<AsRef<str>>`?

Reply:
> @LesnyRumcajs Could you elaborate? Do you suggest using `Vec<AsRef<str>>` or Vec<Url>?

Reply:
Changed to `Url`